waypoint_bag_filename: waypoints.bag
robot_frame: base_link
map_frame: world
robot_pose_topic: /robot_pose
search_angle: 1.57